packages/sitecore-jss-nextjs/package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "@sitecore-jss/sitecore-jss-nextjs",
   "version": "16.0.0-canary.1",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
   "scripts": {
-    "build": "npm run clean && tsc",
+    "build": "npm run clean && tsc -p tsconfig.json && tsc -p tsconfig-esm.json",
```

Comment:
@ambrauer I understand difference between these modules - one from commonjs another from es6.
By default when you import `sitecore-jss-nextjs` it will import code from: `dist/cjs/index.js`, but in which case it will import something from `dist/esm/index.js`? How it's working in our sample?

Reply:
@sc-illiakovalenko The ES modules are used when building through webpack (which is used by Next.js `next build`).